keith-vscode/src/simulation/message.ts

```diff
@@ -23,24 +23,21 @@ export interface WebviewReadyMessage {
     readyMessage: string
 }
 
-export function isWebviewReadyMessage(object: any): object is WebviewReadyMessage {
```

Comment:
Is this no longer needed? What did we use it for?

Reply:
This was an entirely unused function, so I just removed it for cleaning. It was previously used in the old simulation view, that has been replaced with the table-based view.